packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -0,0 +1,110 @@
+import { svg, css, LitElement } from '@lion/core';
+import { localize, LocalizeMixin } from '@lion/localize';
+
+export class LionProgressIndicator extends LocalizeMixin(LitElement) {
+  static get styles() {
+    return [
+      css`
+        :host {
+          display: inline-block;
+        }
+
+        svg.lion-progress {
+          animation: progress-indicator-rotate 2s linear infinite;
+          height: 40px;
+          width: 40px;
+        }
+
+        @keyframes progress-indicator-rotate {
```

Comment:
These keyframes are registered "globally" so there is risk of naming collision with an extension of this component.